### DISCUSSION TITLE
Missing ws_log_defs.h while building Wireshark plugin

### DISCUSSION THREAD

**Discussion by brianheilig** [OpenDDS/OpenDDS#3272] [General]
While following [the instructions to build Wireshark](https://github.com/objectcomputing/OpenDDS/blob/master/tools/dissector/README.md#configure) I encountered the following error:

```
/home/me/wireshark/build/config.h:335:10: fatal error: ws_log_defs.h: No such file or directory
  335 | #include <ws_log_defs.h>
      |          ^~~~~~~~~~~~~~~
```
To resolve this I set the following environment variable before calling 'configure'

`export CPPFLAGS='-I/home/me/wireshark/include'
`

Wireshark version: Version 3.7.0 (v3.7.0rc0-1095-g65e3c4fbe263) 
Configure: ./configure --wireshark-cmake=/home/me/wireshark --wireshark-build=/home/me/wireshark/build --wireshark-lib=run
Open DDS version: 3.19

**Comment by jwillemsen** [comment]
See #2918 for some fixes for wireshark that are currently made

**Reply by iguessthislldo** [reply]
Yea, but a fix for this isn't included in that PR. The focus was on a change in glib. I'm building with the master branch of Wireshark right now to try to recreate the problem though.

**Reply by iguessthislldo** [reply]
And I just got the same exact error. I will push a fix for this in MPC.

**Comment by iguessthislldo** [comment]
Thanks for reporting, I've created a [fix for this in MPC](https://github.com/DOCGroup/MPC/pull/152).
